packages/components/src/components/form/BottomText.tsx

```diff
@@ -30,30 +30,63 @@ export const Container = styled.div<{ $inputState?: InputState; $isDisabled?: bo
         $isDisabled ? theme.textDisabled : getInputStateTextColor($inputState, theme)};
     ${typography.label}
     animation: ${slideDown} 0.18s ease-in-out forwards;
+    background-color: ${({ $inputState, theme }) =>
+        $inputState === 'info' && theme.backgroundAlertBlueSubtleOnElevation1};
+    border-radius: ${borders.radii.xxs};
+
+    ${({ $inputState, theme }) =>
+        $inputState === 'info' &&
+        `
+        margin: ${spacingsPx.sm} 0;
+        padding: ${spacingsPx.xs} ${spacingsPx.sm};
+        ::before {
+            content: '';
+            position: absolute;
+            top: -${spacingsPx.xs};
+            left: 20px;
+            transform: translateX(-50%);
+            width: 0;
+            height: 0;
+            border-left: 8px solid transparent; 
+            border-right: 8px solid transparent; 
+            border-bottom: 8px solid ${theme.backgroundAlertBlueSubtleOnElevation1}; 
+        }
+        `}
+`;
+
+export const Divider = styled.div`
+    height: inherit;
+    align-self: stretch;
+    width: 2px;
+    background-color: ${({ theme }) => theme.backgroundSurfaceElevation1};
+    margin: -${spacingsPx.xs} ${spacingsPx.xs};
 `;
 
 interface BottomTextProps {
     inputState?: InputState;
     isDisabled?: boolean;
-    icon?: IconName;
     children: ReactNode;
 }
 
-export const BottomText = ({
-    inputState,
-    isDisabled,
-    icon = 'warningCircle',
-    children,
-}: BottomTextProps) => {
+export const BottomText = ({ inputState, isDisabled, children }: BottomTextProps) => {
     const theme = useTheme();
+    const iconMap = {
+        error: 'warningCircle',
+        warning: 'warningCircle',
+        info: 'doubleCheck',
+    } as const satisfies Record<InputState, IconName>;
 
     const iconColor: Color | CSSColor = isDisabled
         ? 'iconDisabled'
         : getInputStateTextColor(inputState, theme);
 
     return (
         <Container $inputState={inputState} $isDisabled={isDisabled}>
-            {icon && <Icon name={icon} size="medium" color={iconColor} />}
+            {inputState && iconMap[inputState] && (
+                <Icon name={iconMap[inputState]} size="medium" color={iconColor} />
+            )}
+
+            {inputState === 'info' && <Divider />}
```

Comment:
This shouldn't be dependent on the same. Also if you need it create BottomText as a custom component or add generic prop `hasDivider`.

Reply:
I see what you mean. I will rather create a new custom component and pass it into BottomText & leave BottomText untouched besides the Warning-like refactoring, so we can get rid of the icon when needed & have default re writable icons.

Reply:
Sounds good 👍